rush/rush/src/actions/RebuildAction.ts

```diff
@@ -102,8 +104,19 @@ export default class RebuildAction extends CommandLineAction {
     this._minimalParameter = this.defineFlagParameter({
       parameterLongName: '--minimal',
       parameterShortName: '-m',
-      description: 'Invokes gulp with the "--minimal" option, which speeds up the build by running the minimal set ' +
-        'of tasks required to produce an executable output'
+      description: 'Invokes the build script with the "--minimal" option, which speeds up the build by running the ' +
+        'minimal set of tasks required to produce an executable output'
+    });
+    this._verboseParameter = this.defineFlagParameter({
+      parameterLongName: '--verbose',
+      parameterShortName: '-v',
+      description: 'Display the logs during the build, rather than just displaying the build status summary'
+    });
+    this._veryVerboseParameter = this.defineFlagParameter({
+      parameterLongName: '--very-verbose',
+      parameterShortName: '-vv',
+      description: 'Invokes the build script with the "--verbose" parameter. This implicity enables the "--verbose" ' +
+        'flag which displays the build logs'
```

Comment:
@nickpape-msft Would anyone ever actually want this?

The purpose of "gulp --verbose" is generally that you are debugging a problem with your Gulpfile.  In that scenario, why would you be bulk-building the entire repo using Rush?  Shouldn't you just type "gulp --verbose" for the buggy project?